### ISSUE TITLE
Quickfix items replace grug-far window on open

### ISSUE THREAD

**Issue by kapral18** [MagicDuck/grug-far.nvim#340]
### What happens

When we save the result from grug-far window into quickfixlist and open the items from the list
they replace the grug far window thus inadvertantly creating a split of content.

### What I expect

Grug far window should remain instact and quickfix item should open in the main window even if focus before was in grug-far

### Config

Using default lazyvim grug-far config https://github.com/LazyVim/LazyVim/blob/d0c366e4d861b848bdc710696d5311dca2c6d540/lua/lazyvim/plugins/editor.lua#L131

**Comment by MagicDuck** [comment]
Good idea! I've made it open the first quickfix item immediately in the "main" window! 😄 
Let me know if you see any issue

**Comment by MagicDuck** [comment]
hmm, I see. I was treating it uniformly with the case when there is no main window, in which case a main window needs to be created with some content. I think we can likely do without `cfirst` though if we already have a new window. I'll take a look later.

**Comment by MagicDuck** [comment]
done 😄
